src/linker/Linker/LinkContext.cs

```diff
@@ -174,9 +174,9 @@ public class LinkContext : IDisposable
 
 		public WarningSuppressionWriter WarningSuppressionWriter { get; private set; }
 
-		public HashSet<uint> NoWarn { get; set; }
+		public HashSet<int> NoWarn { get; set; }
 
-		public Dictionary<uint, bool> WarnAsError { get; set; }
+		public HashSet<int> WarnAsError { get; set; }
```

Comment:
Note that this also needs to be able to treat individual warnings not as errors to override the global warnaserror setting (https://github.com/mono/linker/issues/1356#issuecomment-658303720).

Reply:
Yep, missed that